api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -15,7 +15,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// autogenerated by controller-gen object, do not modify manually
+// Code generated by controller-gen. DO NOT EDIT.
```

Comment:
What is the go lang version in your machine? Lets see why it is changed that comment in your environment

Reply:
go version go1.13.1 darwin/amd64